psamm_import/importer.py

```diff
@@ -225,6 +225,12 @@ def model_medium(model, default_flux_limit):
             continue
 
         if len(equation.compounds) != 1:
+            # Provide warning for exchanges reactions with more than
+            # one compound, they won't be put into the medium definition
+            if (len(equation.left) == 0) != (len(equation.right) == 0):
+                logger.warning('Exchange reaction {} has more than one'
+                               ' compound, so it won\'t be converted to'
+                               ' medium format'.format(reaction.id))
```

Comment:
- In your comment: "exchanges reactions" -> "exchange reactions"
- You have some excess `(` and `)` in the `if` statement
- I would change the message to something like: "Exchange reaction {} has more than one compound; it was not converted to medium compounds"